operator/cmd/main.go

```diff
@@ -163,6 +164,9 @@ func run() error {
 				&coreV1.ConfigMap{}: {
 					Label: cacheLabelSelector,
 				},
+				// XXX: Looks like this doesn't cache storage class requests.
+				// Is it even worth it to cache that information?
+				&storagev1.StorageClass{}: {},
```

Comment:
I don't think we have to worry about StorageClass'es impact on RAM, let's drop this change.

Reply:
My intent was to avoid an API call if possible, not to reduce memory footprint. But since it's going to happen rare anyway (only on the PVC creation), yeah, I think it's fine to not cache it.

Reply:
Types not listed here *are* cached by default.